lib/vdsm/storage/lvmdevices.py

```diff
@@ -137,3 +141,26 @@ def _run_vgimportdevices(vg):
 
     if p.returncode != 0:
         raise cmdutils.Error(cmd, p.returncode, out, err)
+
+
+def _run_check():
+    """
+    Check the devices file. As, according to LVM developers, the behavior of
+    this functionality is not entirely or strictly well defined yet, we don't
+    raise any exception if the check finds issues in devices file, but only
+    log a waring with found issues.
+    """
+    cmd = [LVM, 'lvmdevices', "--check"]
+
+    p = commands.start(
+        cmd,
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE
+    )
+    out, err = commands.communicate(p)
+
+    if p.returncode == 0 and err:
+        log.warning("Found following issues in LVM devices file: %s", err)
+
+    if p.returncode != 0:
+        raise cmdutils.Error(cmd, p.returncode, out, err)
```

Comment:
We have too many commands using start() to get the error. We probably need to change our
comamnds.run() helper to return an object with out and and err on success. But should delay this work.